### PR TITLE
Telephone updates

### PR DESCRIPTION
**Pull request by floresj**
**Notes**
- Updated telephone component to reflect design comp updates
- Externalized telephone strings

### CODE REVIEW

File: src/components/Form/Radio/Radio.jsx [hunk 1]
```diff
@@ -83,12 +84,19 @@ export default class Radio extends ValidationElement {
    * Style classes applied to the wrapper.
    */
   divClass () {
-    let klass = 'eapp-blocks-radio'
+    let klass = ''
+    if (!this.props.native) {
+      klass = 'eapp-blocks-radio'
+    }
```
Comment (bmallred) [inline src/components/Form/Radio/Radio.jsx]: OoooOoooOoo, going `native` eh? Good way to toggle right there

File: src/components/Form/Radio/Radio.jsx [hunk 2]
```diff
@@ -133,6 +141,30 @@ export default class Radio extends ValidationElement {
...
+            value={this.state.value}
+            onChange={this.handleChange}
+            onFocus={this.handleFocus}
+            onBlur={this.handleBlur}
+            onClick={this.handleClick}
+            checked={this.state.checked}
+          />
+          <label htmlFor={id}>
+            {this.state.label}
+          {this.props.children}
```
Comment (adelevie) [inline src/components/Form/Radio/Radio.jsx]: spacing?

File: src/components/Form/Telephone/Telephone.test.jsx [hunk 3]
```diff
@@ -35,21 +34,23 @@ describe('The Telephone component', () => {
...
-      focus: false,
-      valid: false
-    }
-    const component = mount(<Telephone name={expected.name} error={expected.error} focus={expected.focus} valid={expected.valid} domestic={true} />)
-    expect(component.find('input#Domestic-phonetype').length).toEqual(1)
-    expect(component.find('input[name="domestic_first"]').length).toEqual(1)
-    expect(component.find('input[name="domestic_second"]').length).toEqual(1)
-    expect(component.find('input[name="domestic_third"]').length).toEqual(1)
-  })
+  //it('renders sane defaults', () => {
```
Comment (adelevie) [inline src/components/Form/Telephone/Telephone.test.jsx]: Delete?

Comment (floresj) [inline src/components/Form/Telephone/Telephone.test.jsx]: I need to uncomment that. Forgot to add it back while I was debugging something.

File: src/components/Form/Radio/Radio.jsx (hunk 2 above)
Comment (floresj) [inline src/components/Form/Radio/Radio.jsx]: That was resolved with another update
screen shot 2017-02-08 at 7 05 47 am

File: src/components/Form/Telephone/Telephone.test.jsx (hunk 3 above)
Comment (bmallred) [inline src/components/Form/Telephone/Telephone.test.jsx]: @floresj maybe format on save/write?

```vim
" Only indent according to file type rules with JSX files
autocmd BufWritePre *.jsx :normal gg=G
```

File: src/components/Form/Radio/Radio.jsx (hunk 2 above)
Comment (bmallred) [inline src/components/Form/Radio/Radio.jsx]: @floresj maybe format on save/write?

```vim
" Only indent according to file type rules with JSX files
autocmd BufWritePre *.jsx :normal gg=G
```

Comment (floresj) [inline src/components/Form/Radio/Radio.jsx]: Yussss def going to use that. <3 vim

**Pull request merged**